odoo/addons/base/models/ir_actions.py

```diff
@@ -679,24 +679,25 @@ def _stringify_path(self):
             pretty_path.append(field_id.field_description)
         return ' > '.join(pretty_path)
 
-    @api.depends('state', 'webhook_field_ids', 'name')
+    @api.depends('state', 'model_id', 'webhook_field_ids', 'name')
     def _compute_webhook_sample_payload(self):
         for action in self:
             if action.state != 'webhook':
                 action.webhook_sample_payload = False
                 continue
             payload = {
-                'id': 1,
+                '_id': 1,
```

Comment:
please leave the key to `id` (no underscore), see why [here](https://github.com/odoo/odoo/pull/138804#discussion_r1370168102)